tests/ethercat/test_pdo.py

```diff
@@ -321,12 +330,15 @@ def test_start_stop_pdo(connect_to_all_slave):
         assert pytest.approx(servo._tpdo_maps[0].items[0].value, abs=2) == servo.read(
             TPDO_REGISTERS[0]
         )
-    # Check that PDOs can be re-started with the same configuration
-    net.start_pdos()
-    start_time = time.time()
-    while time.time() < start_time + timeout:
-        net.send_receive_processdata()
-    net.stop_pdos()
+    # Re-configure the PDOs and re-start the PDO exchange
+    for servo in servos:
+        servo.remove_rpdo_map(rpdo_map_index=0)
+        servo.remove_tpdo_map(tpdo_map_index=0)
```

Comment:
Why do you reconfigure the PDOs? Do you test that you can simply stop and start PDO with the same configuration?

Reply:
Because there is no difference if we use the previous or a new configuration.